packages/python-packages/apiview-gpt/src/_gpt_reviewer.py

```diff
@@ -29,50 +31,70 @@ class GptReviewer:
     def __init__(self):
         self.llm = AzureChatOpenAI(client=openai.ChatCompletion, deployment_name="gpt-4", openai_api_version=OPENAI_API_VERSION, temperature=0)
         self.output_parser = PydanticOutputParser(pydantic_object=GuidelinesResult)
+
+
         self.prompt_template = PromptTemplate(
-            input_variables=["apiview", "guidelines", "language"],
+            input_variables=["apiview", "guidelines", "language", "extra_comments", "class_list"],
             partial_variables={"format_instructions": self.output_parser.get_format_instructions()},
             template="""
-                Given the following {language} Azure SDK Guidelines:
-                  {guidelines}
-                Verify whether the following code satisfies the guidelines:
+                You are trying to analyze an API for {language} to determine whether it meets the SDK guidelines. We only provide one class at a time right now, but if you need it, here's a list of all the classes in this API:
```

Comment:
Did you see better results by providing all the class names?

Reply:
Sort of. It no longer sees just one client (since it runs class by class) and complains "You should provide a sync and async client!"